WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Foundation/UIColor+DesignSystem.swift

```diff
@@ -14,6 +14,20 @@ public extension UIColor {
             public static let secondary = UIColor(named: DesignSystemColorNames.Foreground.secondary)
             public static let tertiary = UIColor(named: DesignSystemColorNames.Foreground.tertiary)
             public static let quaternary = UIColor(named: DesignSystemColorNames.Foreground.quaternary)
+            public static let success = UIColor(named: DesignSystemColorNames.Foreground.success)
+            public static let warning = UIColor(named: DesignSystemColorNames.Foreground.warning)
+            public static let error = UIColor(named: DesignSystemColorNames.Foreground.error)
+
+            public static var brand: UIColor? {
```

Comment:
Does this need to be optional?

Reply:
I think so. The `UIColor(named:` initializer is optional init.

Ref: https://developer.apple.com/documentation/uikit/uicolor/2877380-init

Reply:
I think we could use this as an exception to unwrap it. Yes, the system cannot guarantee that this color will exist when we pass a name but we can make that guarantee on our side.